manifests/2.4.0/opensearch-dashboards-2.4.0.yml

```diff
@@ -14,6 +14,10 @@ components:
     repository: https://github.com/opensearch-project/notifications.git
     working_directory: dashboards-notifications
     ref: '2.x'
+  - name: observabilityDashboards
+    repository: https://github.com/opensearch-project/observability.git
+    working_directory: dashboards-observability
+    ref: '2.x'
```

Comment:
Need to add Windows to all 2.4 builds

Reply:
@peterzhuamazon Can we add Windows target ourself for 2.4.0 release?

Reply:
@bbarani Should I go ahead platform fields in OpenSearch Dahsboards? I don't see it in any other plugin.

Reply:
@ps48 ignore dashboards for now as I am still testing windows with dashboards.
Thanks.